src/jit/codegenxarch.cpp

```diff
@@ -4135,22 +4193,22 @@ void CodeGen::genCodeForCpObj(GenTreeObj* cpObjNode)
         {
 #ifdef DEBUG
             // If the destination of the CpObj is on the stack
-            // make sure we allocated RCX to emit rep movs{d,q}.
+            // make sure we allocated RCX to emit rep movsp.
```

Comment:
Nit: I'd prefer that we leave the comment as-is in order to reflect the actual instructions generated. Same for any other comments that were updated below.